<launch>

	<!-- Machine -->
	<machine name="localhost" address="localhost" env-loader="/opt/ros/hydro/env.sh"/>
	<arg name="machine" default="localhost" />
	
	<node pkg="terrain_server" type="default_flat_map_server" name="terrain_reward_map" output="screen" machine="$(arg machine)">
		<remap from="topic_output" to="head_mount_asus_xtion/depth_registered/points" />
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="world_frame" type="string" value="odom" />
		<param name="x_min" type="double" value="-0.50" />
		<param name="x_max" type="double" value="0.80" />
		<param name="y_min" type="double" value="-0.35" />
		<param name="y_max" type="double" value="0.35" />
		<param name="resolution" type="double" value="0.01" />
		<param name="height" type="double" value="-0.54" />
	</node>
	
</launch>